src/Unirest/Request.php

```diff
@@ -11,6 +13,14 @@ class Request
     private static $handle = null;
     private static $jsonOpts = array();
     private static $socketTimeout = null;
+    private static $enableRetries = false;            // should we enable retries feature
+    private static $maxNumberOfRetries = 3;           // total number of allowed retries
+    private static $retryOnTimeout = false;           // Should we retry on timeout?
+    private static $retryInterval = 1.0;              // Initial retry interval in seconds, to be increased by backoffFactor
+    private static $maxRetryInterval = 120;           // maximum retry wait time (commutative)
```

Comment:
The variable name is misleading. Name should be maximumRetryWaitTime.

Reply:
updated